src/gui/properties/peerlistwidget.h

```diff
@@ -42,6 +44,15 @@ namespace BitTorrent
 {
     class TorrentHandle;
     class PeerInfo;
+
+    struct PeerEndpoint
```

Comment:
Seems we can move this struct into peerlistwidget.cpp and just forward declare it here.

Reply:
Done.